.github/workflows/publish-release.yml

```diff
@@ -86,3 +86,26 @@ jobs:
         with:
```

Comment:
If possible, it would be nice to have publish to npm and publish site work concurrently rather than serially.

This looks like the npm publishers to approve one env, wait for it to finish, and approve the second one.

Instead they could approve both at the same time.

Reply:
Good point. Fixed!